### ISSUE TITLE
no error reporting

### ISSUE THREAD

**Issue by rvion** [haskelly-dev/Haskelly#29]
what I expect from an IDE is to tell me where are erros, and what are errors

exemple in atom:

![image](https://cloud.githubusercontent.com/assets/2150990/22972278/3cf9a682-f379-11e6-99c7-ff696217bb73.png)

haskelly just crash on error

![image](https://cloud.githubusercontent.com/assets/2150990/22972308/6194eb1e-f379-11e6-8e2d-9395d2eba60a.png)

And never recover: once you have an error in your file, you won't have any more tooltips

**Comment by rvion** [comment]
here is how it looks like right now in haskero

![image](https://cloud.githubusercontent.com/assets/2150990/22978310/808eebf0-f392-11e6-9ed2-3f4b1d6468d7.png)

**Comment by martrik** [comment]
Thanks for letting us know. The linter feature is in our roadmap and we'll try to implement it in the future.

**Comment by moodmosaic** [comment]
>The linter feature is in our roadmap and we'll try to implement it in the future.

I don't think we're talking about _linter_ features per-se. As reported also in https://github.com/haskelly-dev/Haskelly/issues/42, it'd be nice to be able to see the *errors* at least...

**Comment by martrik** [comment]
@moodmosaic Indeed, it would be rather an error reporting feature, including the functionality proposed on #39. I'll keep this issue and close #42 .

**Comment by moodmosaic** [comment]
Yeah, I think that's a fundamental feature to have 😃
